src/payment/checkout/Checkout.jsx

```diff
@@ -73,16 +73,19 @@ class Checkout extends React.Component {
     const {
       intl,
       isFreeBasket,
-      loading,
       isBasketProcessing,
+      loading,
+      loaded,
       isPaymentVisualExperiment,
       paymentMethod,
       submitting,
       orderType,
     } = this.props;
 
     const submissionDisabled = loading || isBasketProcessing;
+    const editable = submitting;
```

Comment:
It's counter-intuitive to me that we would want to disable the form when it is `editable` by its current definition. I think we probably want to come up with a new definition of editable here, or just not use it

Reply:
Thanks for seeing that, at some point I had more than one condition, added `submitting` as props without having the separate variable!